azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/WebAppBase.java

```diff
@@ -1229,6 +1250,20 @@
             @Method
             WithCreate<FluentT> withContainerLoggingDisabled();
         }
+
+        /**
+         * A web app definition stage allowing System Assigned Managed Service Identity to be set.
+         * @param <FluentT> the type of the resource
+         */
+        @Beta(SinceVersion.V1_6_0)
+        interface WithManagedServiceIdentity<FluentT> {
+            /**
+             * Specifies that System Assigned Managed Service Identity needs to be enabled in the web app.
+             * @return the next stage of the web app definition
+             */
+            @Method
+            WithCreate<FluentT> withSystemAssignedManagedServiceIdentity();
```

Comment:
you mean return type `Update<FluentT>`? Same for `withContainerLoggingDisabled()` return type